src/deepforest/preprocess.py

```diff
@@ -177,39 +183,39 @@ def split_raster(annotations_file=None,
                       "from existing in-memory numpy object, as numpy_image=")
 
     if path_to_raster:
-        numpy_image = rasterio.open(path_to_raster).read()
-        numpy_image = np.moveaxis(numpy_image, 0, 2)
+        numpy_image = Image.open(path_to_raster)
+        numpy_image = np.array(numpy_image)
     else:
         if image_name is None:
             raise IOError("If passing a numpy_image, please also specify an image_name"
                           " to match the column in the annotation.csv file")
 
-    # Confirm that raster is H x W x C, if not, convert, assuming image is wider/taller than channels
-    if numpy_image.shape[0] < numpy_image.shape[-1]:
-        warnings.warn(
-            "Input rasterio had shape {}, assuming channels first. Converting to channels last"
-            .format(numpy_image.shape), UserWarning)
-        numpy_image = np.moveaxis(numpy_image, 0, 2)
+    # If its channels last, convert to channels first
+    if numpy_image.shape[2] in [3, 4]:
+        print(
+            "Image shape is {}, assuming this is channels last, converting to channels first"
+            .format(numpy_image.shape[2]))
+        numpy_image = numpy_image.transpose(2, 0, 1)
 
     # Check that it's 3 bands
     bands = numpy_image.shape[2]
     if not bands == 3:
         warnings.warn(
-            "Input rasterio had non-3 band shape of {}, ignoring "
-            "alpha channel".format(numpy_image.shape), UserWarning)
+            "Input image had non-3 band shape of {}, selecting first 3 bands".format(
+                numpy_image.shape), UserWarning)
         try:
-            numpy_image = numpy_image[:, :, :3].astype("uint8")
+            numpy_image = numpy_image[:3, :, :].astype("uint8")
         except:
             raise IOError("Input file {} has {} bands. "
                           "DeepForest only accepts 3 band RGB rasters in the order "
-                          "(height, width, channels). "
+                          "(height, width). "
```

Comment:
This error message is confusing to me because it's talking about 3 bands in an order and then it's talking about height and width, not bands.